src/main/java/com/microsoft/azure/servicebus/jms/ServiceBusJmsConnectionFactory.java

```diff
@@ -333,4 +376,15 @@ private void ensureInitialized() {
             throw new RuntimeException("This ServiceBusJmsConnectionFactory object is not initialized with the proper parameters.");
         }
     }
-}
+    
+    void setExtensionsForAad() {
```

Comment:
>void [](http://example.com/codeflow?start=4&length=4)

nit: may be call it private since we are explicitly specifying elsewhere as well